### PR TITLE
Status command

### PR DESCRIPTION
**Pull request by otter-computer**
Adds the `!Status`command.
- Displays the time the bot was deployed
- Shows the latest commit
- Has a link to the commit page on GitHub
- Only useable by members with the `Bot Developer` role

### CODE REVIEW

**Comment by chrislennon** [comment]
👍 
Reference #146 - using [option 1](https://github.com/gaymers-discord/DiscoBot/issues/146#issuecomment-268479040) seems to be working after commits have taken place.

**Review by brandon-a — APPROVED** [review]

File: commands/status.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  usage: '',
+  description: 'BOT DEVELOPER ONLY: Shows the latest deploy from GitHub.',
+  allowDM: false,
+  requireRoles: ['Bot Developer'],
+  process: (bot, message) => {
+    message.reply(
+      'Bot deployed at: `' + process.env.HEROKU_RELEASE_CREATED_AT + '`\n' +
+      'https://github.com/gaymers-discord/DiscoBot/commit/' + process.env.HEROKU_SLUG_COMMIT + '\n' +
```
Comment (jmariondev) [inline commands/status.js]: Maybe wrap on 80 here?

**Review by jmariondev — CHANGES_REQUESTED** [review]
Maybe wrap on 80 with this code?

Also, it would be good to integrate #157 into this so the command can only be used in the bot dev room.

Other than that, LGTM.

**Comment by otter-computer** [comment]
- Added onlyIn prop
- Truncated message lines to 80 (ish, I did this on github web editor don't hurt me)

**Review by jmariondev — APPROVED** [review]

**Pull request merged**